examples/grpc/src/main/java/io/opentelemetry/example/grpc/HelloWorldClientStream.java

```diff
@@ -163,6 +162,19 @@ public void start(Listener<RespT> responseListener, Metadata headers) {
     }
   }
 
+  private static OpenTelemetry initTracing(LoggingSpanExporter exporter) {
+    // install the W3C Trace Context propagator
+    // Get the tracer management instance
+    SdkTracerProvider sdkTracerProvider = SdkTracerProvider.builder().build();
```

Comment:
Add a TODO to use the new builder option `addTracerProvider`

Reply:
I assume you mean `addSpanProcessor()`. Yes, we'll need to do this after we publish the next release, but I didn't want to confuse users looking at our docs with TODOs from unreleased changes.

Reply:
K